### PR TITLE
Prep for 0.7.0 release

### PR DESCRIPTION
**Pull request by adleong**

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-## x.x.x
+## 0.7.0
```
Comment (olix0r) [inline CHANGES.md]: leave ## x.x.x up top?

**Comment by olix0r** [comment]
:star:

File: CHANGES.md (hunk 1 above)
Comment (adleong) [inline CHANGES.md]: We haven't done this for previous releases:
https://github.com/BuoyantIO/linkerd/blob/0.6.0/CHANGES.md
https://github.com/BuoyantIO/linkerd/blob/0.5.0/CHANGES.md

Comment (adleong) [inline CHANGES.md]: oh, we did for 0.4.0... but just that one 😬

**Comment by rmars** [comment]
Suggestions for the dashboard changes (wordsmithing appreciated):

```
* Admin site/dashboard UI improvements:
  * Added server success rate graphs to the dashboard, improved responsiveness
  * Added the ability to navigate to a specific router's dashboard
  * Standardized the look and feel of the admin pages
```

**Pull request merged**